packages/xrpl/test/integration/fundWallet.test.ts

```diff
@@ -150,4 +151,26 @@ describe('fundWallet', function () {
     },
     TIMEOUT,
   )
+
+  it('handles errors', async () => {
+    const api = new Client('wss://s.altnet.rippletest.net:51233')
+    await api.connect()
+
+    // jasmine and jest handle async differently so need to use try catch approach instead of `expect.rejects` or `expectAsync`
+    try {
+      await api.fundWallet(null, {
+        amount: '-1000',
+        usageContext: 'integration-test',
+      })
+
+      throw new Error('Error not thrown')
+    } catch (error) {
+      await api.disconnect()
+      expect(error).toEqual(
+        new XRPLFaucetError(
+          'Request failed: {"body":{"error":"Invalid amount","detail":"Must be an integer"},"contentType":"application/json; charset=utf-8","statusCode":400}',
```

Comment:
nit: can't be fixed here but shouldn't the message say "Must be an positive integer"?

Reply:
This is the response that comes back from the faucet server.  This test gets the live response from the testnet faucet.